### ISSUE TITLE
Requesting more examples of standlone links in context on the Link page

### ISSUE THREAD

**Issue by tomlroach** [carbon-design-system/carbon-website#3869]
## Summary
I'd like to request a little more detail on the link page, specifically related to the section on standalone links, and perhaps an update to one of the images to clarify some details. 

## Relevant information

The difference between inline and standalone links is described on the links page [here](https://carbondesignsystem.com/components/link/usage/)

However, only an inline link is ever shown in context. All the images for standalone links show only the link styling, and not it's usage. As a result, I find a little confusion about whether to use an inline link or a standalone link "following" a paragraph of text. 
This is a common example I see where a standalone link is used because it is "following" the text, and not part of the sentence. 
<img width="702" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/6569324/674ba837-99ef-477f-aabf-58d8bff4aada">

My interpretation is that this should probably be using an inline link _or_ the "Learn more" should be placed on a new line, and then it would use a standalone link styling. But because there are no examples in the page of using a standalone link it's left unclear. 

I'd suggest adding one or more additional images to this section showing some examples of Standalone link usage. 
https://carbondesignsystem.com/components/link/usage/#standalone-link

**Comment by tomwaterton** [comment]
@thyhmdo has asked me to have a look at this issue and add any feedback I have.

Having just read the existing guidance for the [Link](https://carbondesignsystem.com/components/link/usage) component, I agree with @tomlroach’s interpretation that the following design would be incorrect:

> _Incorrect design example:_
>
> ![Screenshot 2024-01-29 at 15 54 08](https://github.com/carbon-design-system/carbon-website/assets/9316771/0e6371f7-10aa-47c5-a315-273f41ebb617)

**Rationale**
- Because the link text is part of the paragraph of text, the _inline link_ variant should be used (which is styled with an underline to make it stand out from the text around it).

I also think that these examples would be incorrect: 

> _Incorrect design examples:_
>
> ![Screenshot 2024-01-29 at 16 01 14](https://github.com/carbon-design-system/carbon-website/assets/9316771/0687a691-6a32-4ad8-ab3c-e2742db1140b)

**Rationale**
- In the first case: because the link text is still part of the paragraph, the _inline link_ variant should be used (which is styled with an underline). Also, according to the current guidelines, an icon should not be used here.
- In the second case: I don't think there is nearly enough vertical separation between the paragraph of text and the standalone link (which by definition is not part of the paragraph but is meant to stand alone separate from other elements). 


### Valid examples
I also agree that based on the current guidelines, either of the two choices shown in @tomlroach ’s following examples would be valid (and the second example could optionally be paired with an icon): 

> _correct design examples:_
>
> ![Screenshot 2024-01-29 at 15 54 16](https://github.com/carbon-design-system/carbon-website/assets/9316771/d44a1abd-375f-4ee8-b21f-b556d3ad7743)


### Suggested enhancements that would further improve this guidance: 
1. Near the top of the current guidance, it states that inline links "should not be paired with an icon". I understand that this true in general (and a distinction between inline links and standalone links). However, I wonder if we should qualify this text to explicitly state that the launch icon is an exception to this rule. 
![Screenshot 2024-01-29 at 17 34 10](https://github.com/carbon-design-system/carbon-website/assets/9316771/e34fb767-8806-423f-a804-37ce4a796381)

1. We should specify the vertical padding needed between the bottom of a paragraph and the top of a standalone link. (I am not a visual designer so cannot advise the specific pixel value, but I would expect at least one line’s gap between the two elements.)

1. There is already one product UI example that shows a couple of inline links. I think this would be even more helpful if it showed an inline link mid-sentence and an inline link that comes at the end of the paragraph. (This would make it clearer to all that even if the link comes at the end of the paragraph, if it is still part of the paragraph then the _inline link_ variant should be used.)

1. It is fairly common in IBM products to see several "Learn more" links used within the same context (e.g. one for each tile on a product's homepage). I know that the existing **Accessibiliy** tab includes the following guidance, but I wonder whether it’s worth repeating this on the main **Usage** tab as well, as this is something that in my experience many designers are not aware of, and I worry that many will refer to the Usage tab and not bother to look at the Accessibility tab: 
> ![Screenshot 2024-01-29 at 16 14 43](https://github.com/carbon-design-system/carbon-website/assets/9316771/b6eeca72-6bc9-43ef-87d4-ccde5131ae27)

**Comment by mbgower** [comment]
>the inline link variant should be used (which is styled with an underline to make it stand out from the text around it

Despite the text in the Usage doc, it's important to note that there are situations where Carbon's inline link style suppresses underlines, even on the Link usage page
![image](https://github.com/carbon-design-system/carbon-website/assets/14298245/d9bdd944-79ed-489c-87ae-d7690034bd1f)

There is some irony to me that Carbon supresses the underline when the link is _not_ inline, because the predominant argument I've heard for removing an underline on a link is when there are so many inline, it disrupts the reading of the content. Once the links are not within a text context, it seems equally important to me to provide visual reinforcement that this chunk of text _does_ something, and what that may be.

That said, I overall agree with the takeaways and interpretations of what is and is not inline in this issue.

**Comment by thyhmdo** [comment]
thank you @tomwaterton and @mbgower these are great points and we did run into a discussions about a few things you mentioned. 

On number 4 that @tomwaterton mentioned, does it work the same in this format? (View all and Manage users) 
![image](https://github.com/carbon-design-system/carbon-website/assets/77697351/874704b2-275c-4575-817f-6eebc8b18580)

**Comment by mbgower** [comment]
> On number 4 that Tom mentioned, does it work the same in this format? (View all and Manage users)

Yes, it works the same, and I think the space constraint here (which doesn't really exist when one it just shoving a link at the bottom of a card) gives more weight to the need for a concise link name (as opposed to [View planned maintenance](url))

It would sure be cool if we could have an out-of-the-box pattern where any use of a number of generic link names were associated by the title value of the parent object -- or at least prompted the designer for that -- and then gracefully incorporated with an aria-describedby
